.github/workflows/js-getsentry-dispatch.yml

```diff
@@ -33,19 +31,32 @@ jobs:
           app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
           private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
 
-      # Notify getsentry only if there were frontend files changed
       - name: Dispatch getsentry frontend tests
         uses: actions/github-script@v3
         with:
           github-token: ${{ steps.getsentry.outputs.token }}
           script: |
+            // Check for getsentry PR dependency
```

Comment:
Is it possible to pull this into a file so it's not so.. inline?

Reply:
It is, I'd prefer to get this all working before refactoring though